tokio-console/src/config.rs

```diff
@@ -320,7 +326,9 @@ impl Config {
     }
 
     pub(crate) fn retain_for(&self) -> Option<Duration> {
-        self.retain_for.as_ref().and_then(|value| value.0)
+        self.retain_for
+            .unwrap_or_default()
+            .0
```

Comment:
hmm, looks like `rustfmt` would prefer to see
```suggestion
        self.retain_for.unwrap_or_default().0
```
here: https://github.com/tokio-rs/console/actions/runs/3202161732/jobs/5240217564